### PR TITLE
Added Unique_Number.cpp and Updated REAMDME.md

### PR DESCRIPTION
**Pull request by Fab-Gaurav**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

Given an integer array , finding the Unique Elements in the array that appears exactly once and return the sum of the unique element array.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

https://github.com/TesseractCoding/NeoAlgo/issues/4094

### CODE REVIEW

File: C-Plus-Plus/math/Unique_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+OUTPUT : 9
+
+Explanation : Unique elements are [1, 2, 6] and the Sum is 9
+
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main(){
```
Comment (HarshCasper) [inline C-Plus-Plus/math/Unique_Number.cpp:17]: Don't put your code logic in `main()` function. Use `main()` for the driver code

File: C-Plus-Plus/math/Unique_Number.cpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/*
+
+You are given an integer array nums. The unique elements of an array are 
+the elements that appears exactly once in the array. Return the sum of all 
+Unique elements of nums.
+
+INPUT : [1, 5, 2, 6, 5]
+OUTPUT : 9
+
```
Comment (HarshCasper) [inline C-Plus-Plus/math/Unique_Number.cpp:9]: Keep sample I/O at the end of the code

File: C-Plus-Plus/math/Unique_Number.cpp [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    }        
+    
+    if(ctr==0){
+      cout<<arr1[i]<<",";
+      sum+=arr1[i];
+     }
+    }    
+    cout<<endl<<"Sum of Unique elements : "<<sum;
+    return 0;
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/math/Unique_Number.cpp:47]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Fab-Gaurav** [comment]
@HarshCasper I have made the required changes.
Have a look : https://github.com/TesseractCoding/NeoAlgo/pull/4270

**Review by ankitaggarwal23 — COMMENTED** [review]
@Fab-Gaurav Don't open new PR for each changes requested by @HarshCasper. Closing all your PR open a new one Do all the changes Harsh requested.

**Comment by Fab-Gaurav** [comment]
> @Fab-Gaurav Don't open new PR for each changes requested by @HarshCasper. Closing all your PR open a new one Do all the changes Harsh requested.

Done @ankitaggarwal23 
I have make the changes and make the new PR #4400
Please have a look @HarshCasper 
Thank you

**Pull request closed without merging**